src/components/Contribute/Skill/ReviewSubmission/index.tsx

```diff
@@ -2,85 +2,107 @@
 import { SkillFormData } from '@/types';
 import { Content, ContentVariants } from '@patternfly/react-core';
 import React from 'react';
+import './submission.css';
+import { Accordion, AccordionContent, AccordionItem, AccordionToggle } from '@patternfly/react-core';
 
 interface ReviewSubmissionProps {
   skillFormData: SkillFormData;
   isGithubMode: boolean;
 }
 
 export const ReviewSubmission: React.FC<ReviewSubmissionProps> = ({ skillFormData, isGithubMode }) => {
+  const [expanded, setExpanded] = React.useState<{ [key: string]: boolean }>({});
+
+  const onToggle = (id: number) => {
+    setExpanded((prevState) => ({
+      ...prevState,
+      [id]: !prevState[id]
+    }));
+  };
+
   return (
-    <div>
-      <Content component={ContentVariants.h2}>Review Submission</Content>
-      {/* Author Information */}
-      <p>
-        <strong>Author Information</strong>
-      </p>
-      <p>
-        <i>Name:</i> {skillFormData.name}
-      </p>
-      <p>
-        <i>Email:</i> {skillFormData.email}
-      </p>
+    <>
+      <section className="review-submission-container">
+        <Content component={ContentVariants.h3}>Review</Content>
+        <p>Review the information below and click finish to submit your knowledge contribution. Use the back button to make changes.</p>
```

Comment:
This is skill review form, so we need to remove all the reference for "Knowledge" through out this form.